src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+Title: Back up and restore Juju
+
+# Backing up And Restoring Juju
+
+It is always a good idea to keep backups, and it is possible to backup both the 
```

Comment:
Personally, I use "back up" when referring to the act of backing up and "backup" as the result of backing up.